docker/Dockerfile.dhtnode

```diff
@@ -0,0 +1,17 @@
+ARG DIST=xenial
+FROM sociomantictsunami/dlang:$DIST-v5 as builder
+COPY . /project/
+WORKDIR /project/
+ARG DIST=xenial
```

Comment:
This `ARG DIST=xenial` is there twice on purpose?

Reply:
Yes: any `ARG` defined before `FROM` line will only have effect until that line and needs to be reintroduced after.